### ISSUE TITLE
Suggestion - Trueskill rankings

### ISSUE THREAD

**Issue by shadowfox87** [cvantum/discord-bot#10]
This is the first Discord bot on the net that uses the challonge api. I was wondering if you could take it one step further and calculate Trueskill ratings? Here is a C# program that is able to calculate Trueskill ratings using the challonge API: https://github.com/SugoiFactory/SkillKeeper
Maybe you can convert that into node js?

**Comment by cvantum** [comment]
I'll take a look at the project. Maybe i can implement it. Most API-Features are depending on the User who created the API-Key. You have only (an easier) access to the tournaments which the user has created. Otherwise you have to select every tournament on own like i did with this Bot.

I have to rewrite the code with the recommandation to use a API-Key from the user who created the whole tournaments. And then i hope i can select singe user from a team too :-)

**Comment by shadowfox87** [comment]
Yes, the user should be able to enter their api key in pm with the bot. Then they can start a tournament in public channel with specific parameters (Single Elim, start date, etc.). The bot will use the api key of the user who told it to start tournament. If no api key is registered for that user, the bot will ask that api key must be entered in pm.

**Comment by cvantum** [comment]
This Bot isn't made to run on serveral servers with multiple data :-) So i think i have to make some redesign to make this work. But i'll keep this in my mind and would integrate this feature first, also with several api-keys from different users. 

You can also ask me further questions on Discord. My Name is: cvantum#0867. You can find me on https://discord.gg/discord-api

Honestly to say.. this bot was my first test and my first scipt in Node.JS i have a lot of stuff to do to rebuild this bot with ECMA6 :-) But i will make some updates and let you know it if this bot will be work with your wish.

**Comment by shadowfox87** [comment]
Thanks.

**Comment by shadowfox87** [comment]
Thanks and actually you should instead just look at the challonge api: http://api.challonge.com/v1
Currently there's no code in node js. The first link I posted above is probably not the best one to use since that is a program used to calculate Trueskill ratings. I'm just looking for a bot that uses challonge api which you've attempted.
